hmmlearn/hmm.py

```diff
@@ -554,12 +555,31 @@ def __init__(self, n_components=1, n_mix=1,
         self.covariance_type = covariance_type
         self.min_covar = min_covar
         self.n_mix = n_mix
+        self.weights_prior_alphas = np.asarray(weights_prior)
```

Comment:
We try to follow `sklearn` [API conventions](http://scikit-learn.org/stable/developers/contributing.html#apis-of-scikit-learn-objects) in `hmmlearn`

> [...] every keyword argument accepted by __init__ should correspond to an attribute on the instance.